Formula/f/fastfec.rb

```diff
@@ -28,7 +36,35 @@ class Fastfec < Formula
     depends_on "pcre" # PCRE2 issue: https://github.com/washingtonpost/FastFEC/issues/57
   end
 
+  resource "zig" do
+    url "https://ziglang.org/download/0.11.0/zig-0.11.0.tar.xz"
+    sha256 "72014e700e50c0d3528cef3adf80b76b26ab27730133e8202716a187a799e951"
+  end
+
   def install
+    resource("zig").stage do
+      ENV["NIX_LDFLAGS"] = ENV["HOMEBREW_RPATH_PATHS"].split(":").map { |p| "-rpath #{p}" }.join(" ") if OS.linux?
+
+      cpu = case Hardware.oldest_cpu
+      when :arm_vortex_tempest then "apple_m1" # See `zig targets`.
+      else Hardware.oldest_cpu
+      end
+
+      args = ["-DZIG_STATIC_LLVM=ON"]
+      args << "-DZIG_TARGET_MCPU=#{cpu}" if build.bottle?
```

Comment:
```suggestion
      args = ["-DZIG_STATIC_LLVM=ON"]
```
Since this is build-only.